tests/python/mkl/test_mkldnn.py

```diff
@@ -441,7 +440,4 @@ def backward(self, req, out_grad, in_data, out_data, in_grad, aux):
     custom = mx.symbol.Custom(name='custom', data=conv, op_type='custom')
     exec1 = custom.bind(mx.cpu(), args={'data': mx.nd.ones([10,3,96,96]), 'conv_weight': mx.nd.ones([8,3,5,5])})
     exec1.forward()[0].wait_to_read()
-
-
-if __name__ == '__main__':
-    install.test_mkldnn_install()
```

Comment:
Need make sure about the purpose of a test case before removing it. If this case is just for verifying whether MKL-DNN so is correctly linked, I'm OK with removing it. Otherwise, need replace it with something else.

Reply:
The test looks if mkldnn appears on the memory table of the process running mxnet.I have verified manually (by `cat /proc/{id}/maps | grep mkldnn`) that mkldnn use to appear in the table when it is dynamically linked but no longer does (statically linked).

the second part of this just checks the response code (rc - probably to make sure the proc/{id}/maps file even exists).

@pengzhao-intel is there another reason this test was added